tests/rptest/tests/redpanda_oauth_test.py

```diff
@@ -127,11 +162,14 @@ def get_client_credentials_token(self, cfg):
             client_id=cfg.client_id,
             client_secret_key=cfg.client_secret,
             realm_name=DEFAULT_REALM,
-            verify=True)
+            verify=False)
```

Comment:
Is this required now that the certificate is "installed" with `update-ca-certificates`?

Reply:
This request originates from the "driver" (?) node, so yes. Better approaches include:
- don't use the `python-keycloak` for this (construct the API call by hand and pass the CA cert to `requests`)
- modify `python-keycloak` to accept a CA cert here (like `requests`) instead of just bool

Neither of which seems particularly worth the effort, under the circumstances.